language/move-lang/src/parser/syntax.rs

```diff
@@ -964,19 +1009,22 @@ fn parse_name_exp(context: &mut Context) -> Result<Exp_, Diagnostic> {
     let n = parse_name_access_chain(context, || {
         panic!("parse_name_exp with something other than a ModuleAccess")
     })?;
+    parse_name_exp_after_access_chain(context, n)
+}
 
+fn parse_name_exp_after_access_chain(
```

Comment:
I don't see this `parse_name_exp_after_access_chain` function used elsewhere. Am I missing something or did you split it just to keep the functions smaller?

Reply:
I think I split it for an older version of the parser and didn't unsplit it